openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -173,7 +177,10 @@ <h4 class="subsection-title">
                                               </div>
                                             </div>
                         % if vertical.get('complete'):
-                                                <span class="complete-checkmark fa fa-check"></span>
+                                                <span class="complete-checkmark fa fa-check">
+                                                    <span class="sr">${_("Completed")}</span>
```

Comment:
Could you please put the class="sr" spans outside the checkmark span, and add aria-hidden="true" to the checkmark span (as with the file above)?  As you have it now, the checkmark may be read by some screen readers.